packages/components-storybook-v2/test/integration/Loaders.test.js

```diff
@@ -0,0 +1,20 @@
+describe('Icons', () => {
```

Comment:
Icons? Probably Loader right?

Reply:
yup it should be Loader. fixed it, need to update snapshots. adding in progress label...